### PR TITLE
wolfssl: enable debug logging in wolfSSL when compiled in

### PR DESCRIPTION
**Pull request by julek-wolfssl**

### CODE REVIEW

File: src/openssl.c [hunk 1]
```diff
@@ -1101,6 +1101,9 @@ void _libssh2_openssl_crypto_init(void)
     ENGINE_register_all_complete();
 #endif
 #endif
+#ifdef DEBUG_WOLFSSL
```
Comment (vszakats) [inline src/openssl.c:1104]: Let's also guard it with `LIBSSH2_WOLFSSL`.

Comment (vszakats) [inline src/openssl.c:1104]:
```suggestion
#if defined(LIBSSH2_WOLFSSL) && defined(DEBUG_WOLFSSL)
```

Comment (julek-wolfssl) [reply src/openssl.c]: Added `LIBSSH2_WOLFSSL` to macro check.

Comment (vszakats) [reply src/openssl.c]: Thanks @julek-wolfssl! and merging.

**Pull request merged**